src/redux/reducers/authSlice.ts

```
@@ -1,109 +1,90 @@
+  void,
   { email: string; password: string },
   { rejectValue: string }
 >("/auth/login", async (payload, { rejectWithValue }) => {
   try {
     const res = await postLogin(payload);
     if (res.pass) {
-      const { access_token, refresh_token }: AuthResponse = res.data;
-      return { access_token, refresh_token };
+      return;
     } else return rejectWithValue(res.data as string);
   } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)